questions/qa-byte-order-mark.en.html

```diff
@@ -104,7 +104,8 @@ <h2>Answer</h2>
 <h3> What is a byte-order mark?</h3>
 
 <div class="sidenoteGroup">
-<p>At the beginning of a page that uses a <a class="termref" href="/International/articles/definitions-characters/#unicode">Unicode</a> <a class="termref" href="/International/articles/definitions-characters/#charsets">character encoding</a> you may find some bytes that represent the Unicode code point U+FEFF BYTE ORDER MARK (abbreviated as <dfn>BOM</dfn>).</p>
+<p>A <dfn>Byte Order Mark (BOM)</dfn> is a special <a class="termref" href="/International/articles/definitions-characters/#unicode">Unicode</a> character (U+FEFF) that can appear at the very beginning of a text file. Its primary original purpose was to indicate the byte order (or <a href="https://en.wikipedia.org/wiki/Endianness">endianness</a>) for 16-bit and 32-bit Unicode <a class="termref" href="/International/articles/definitions-characters/#charsets">encodings</a>. While often invisible and intended to aid in correctly interpreting text, the BOM can sometimes cause unexpected display issues or problems with software if not handled correctly, particularly with UTF-8 encoded files where its use is not for endianness.</p>
```

Comment:
A lot of things seem crammed into one paragraph. Plus there might be some missing info? Perhaps:

```suggestion
<p>A <dfn>Byte Order Mark</dfn>, sometimes abbreviated "BOM", is a special Unicode character intended to appear at the very beginning of a text file. Its original purpose was to indicate the <q><a href="https://en.wikipedia.org/wiki/Endianness">endianness</a></q> of text that used the UTF-16 or UTF-32 character encodings of Unicode. The Byte Order Mark is <kbd>U+FEFF ZERO WIDTH NON-BREAKING SPACE</kbd>: the character name refers to a separate, deprecated, use of the character.</p>
<p>Some systems use the BOM code point at the start of a file to indicate that text files are using the UTF-8 character encoding, even though UTF-8 does not need a marker to indicate endianness.</p>
<p>While often invisible and intended to aid in correctly interpreting text, the presence of the BOM can sometimes cause unexpected display issues or problems with software if not handled correctly.</p>
```

Reply:
Thanks. I adjusted the opening text.